diff --git a/node_modules/@docusaurus/theme-search-algolia/lib/theme/SearchBar/index.js b/node_modules/@docusaurus/theme-search-algolia/lib/theme/SearchBar/index.js
index 878dc9b..8393b9b 100644
--- a/node_modules/@docusaurus/theme-search-algolia/lib/theme/SearchBar/index.js
+++ b/node_modules/@docusaurus/theme-search-algolia/lib/theme/SearchBar/index.js
@@ -4,19 +4,19 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-import React, {useState, useRef, useCallback, useMemo} from 'react';
-import {createPortal} from 'react-dom';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
-import {useHistory} from '@docusaurus/router';
-import {useBaseUrlUtils} from '@docusaurus/useBaseUrl';
-import Link from '@docusaurus/Link';
+import { DocSearchButton, useDocSearchKeyboardEvents } from '@docsearch/react';
 import Head from '@docusaurus/Head';
-import {isRegexpStringMatch} from '@docusaurus/theme-common';
-import {useSearchPage} from '@docusaurus/theme-common/internal';
-import {DocSearchButton, useDocSearchKeyboardEvents} from '@docsearch/react';
-import {useAlgoliaContextualFacetFilters} from '@docusaurus/theme-search-algolia/client';
+import Link from '@docusaurus/Link';
+import { useHistory } from '@docusaurus/router';
+import { isRegexpStringMatch } from '@docusaurus/theme-common';
+import { useSearchPage } from '@docusaurus/theme-common/internal';
+import { useAlgoliaContextualFacetFilters } from '@docusaurus/theme-search-algolia/client';
 import Translate from '@docusaurus/Translate';
+import { useBaseUrlUtils } from '@docusaurus/useBaseUrl';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import translations from '@theme/SearchTranslations';
+import React, { useCallback, useMemo, useRef, useState } from 'react';
+import { createPortal } from 'react-dom';
 let DocSearchModal = null;
 function Hit({hit, children}) {
   return <Link to={hit.url}>{children}</Link>;
@@ -110,8 +110,9 @@ function DocSearch({contextualSearch, externalUrlRegex, ...props}) {
       if (isRegexpStringMatch(externalUrlRegex, item.url)) {
         return item;
       }
-      // We transform the absolute URL into a relative URL.
-      const url = new URL(item.url);
+      // We transform the absolute URL into a relative URL. (patch we remove /docs/ from the url to make it work with preview deploys)
+      const isPreviewUrl = window.location.href.match('https:\\/\\/docusaurus-docs-.*-statelyai\\.vercel\\.app')?.length > 0
+      const url = new URL(isPreviewUrl ? item.url.replace('/docs/', '/') : item.url);
       return {
         ...item,
         url: withBaseUrl(`${url.pathname}${url.hash}`),
